apps/docs/pages/guides/storage/image-transformations.mdx

```diff
@@ -149,6 +175,34 @@ supabase.storage.from('bucket').download('image.jpg', {
 | BMP    | `bmp`     | ☑️     | ☑️     |
 | TIFF   | `tiff`    | ☑️     | ☑️     |
 
+## Self Hosting
+
+Our solution to image resizing and optimisation can be self-hosted as with any other Supabase product.
+Under the hood we use the awesome [Imgproxy](https://imgproxy.net/)
+
+#### Imgproxy Configuration:
```

Comment:
This guide is for the hosted platform. Can we move the imgproxy configuration to the [self hosting section](https://supabase.com/docs/guides/self-hosting/storage/config) instead?

Reply:
I think i can add the configuration in the section you mentioned, but i think it would still be nice to showcase how simple could be to self host it